src/Entities/Entity.cpp

```diff
@@ -158,19 +157,28 @@ bool cEntity::Initialize(OwnedEntity a_Self, cWorld & a_EntityWorld)
 
 	cPluginManager::Get()->CallHookSpawnedEntity(a_EntityWorld, *this);
 
+	return true;
+}
+
+
+
+
+
+void cEntity::OnAddToWorld(cWorld & a_World)
```

Comment:
Update m_LastSentPosition here, so entities spawn at the right location client-side.